e2e/testsuite/testsuite.go

```diff
@@ -154,6 +154,10 @@ func (s *E2ETestSuite) ConfigureRelayer(ctx context.Context, chainA, chainB ibc.
 	pathName := s.generatePathName()
 
 	channelOptions := ibc.DefaultChannelOpts()
+	// TODO: better way to do this.
+	// For now, set the version to the latest transfer module version
```

Comment:
think this was just because `ibc.DefaultChannelOpts()` defaults to `ics20-1` so we are overriding that with `ics20-2`, this actually isn't too bad, we could also leave it as `empty` string but I think I'd prefer the explicit override here.

Reply:
I see. This is because `ibc.DefaultChannelOpts()` is used by the grandpa wasm test. We could optionally change that code to use [`s.TransferChannelOpts()`](https://github.com/cosmos/ibc-go/blob/8eae033732183be839b3d368a6b97720768e7b3b/e2e/testsuite/testsuite.go#L454)?

Reply:
ah function signature mismatch. `DefaultChannelOpts()` returns a type whereas TransferChannelOptions returns a func

Reply:
https://github.com/cosmos/ibc-go/pull/6360